core/src/main/java/me/leoko/advancedban/utils/Regex.java

```diff
@@ -0,0 +1,147 @@
+package me.leoko.advancedban.utils;
+
+import java.util.regex.Pattern;
+
+/**
+ * @author <a href="https://github.com/iGabyTM">GabyTM</a>
+ */
+public enum Regex {
+
+    ZERO_OR_N_DIGITS("[0-9]*"),
```

Comment:
From what I see, this regex can be replaced with the `DIGITS` regex, as the case of no digits is irrelevant in the places where it is used